service/rpi-sb-provisioner.sh

```diff
@@ -497,6 +509,55 @@ if [ -z "${DEMO_MODE_ONLY}" ] && [ -n "${RPI_DEVICE_FETCH_METADATA}" ]; then
     esac
 
     keywriter_log "Board is: ${BOARD_STR}, with revision number ${REVISION}. Has Processor ${PROCESSOR_STR} with Memory ${MEMORY_STR}. Was manufactured by ${MANUFACTURER_STR}"
+
+    if [ -f "${RPI_SB_PROVISONER_MANUFACTURING_DB}" ]; then
```

Comment:
How is it created in the first place?

Reply:
sqlite automatically creates the backing file, if it does not exist.

Reply:
sqlite isn't invoked outside this 'if [ -f ]' block though? The backing file does not exist, so sqlite is not invoked? Should this be 'if [ ! -z ]' or similar?

Reply:
Not convinced that would work - among other things, that might lead to a multi-content database, or an existing database becoming a WAL object.

Probably the safest thing to do is to let this error out, and include guidance that a pre-processing step is required.